README.md

```diff
@@ -22,10 +22,14 @@ The VM should satisfy the following requirements:
 * Prepare a Windows 10+ virtual machine
   * Install Windows in the virtual machine, for example using the raw Windows 10 ISO from https://www.microsoft.com/en-us/software-download/windows10ISO
   * Ensure the [requirements above](#requirements) are satisfied, including:
-    * Disable Windows Updates (at least until installation is finished)
-      * https://www.windowscentral.com/how-stop-updates-installing-automatically-windows-10
```

Comment:
what is the problem with these instructions?

Reply:
They don't work anymore apparently - https://twitter.com/0xdf_/status/1725287455694163988

Reply:
At least they didn't work for me. Using GPOs would require a reboot, and on reboot, it would be back and running again.

Reply:
@0xdf223 are you using Windows 10? Because I think I used that guide few weeks ago and it worked. 🤔

Reply:
Yes, win 10. Are you fully updated before trying to do this?

Maybe the best option would be to leave the old one and add the new references? I'm not the first person to run into this issue: https://github.com/mandiant/flare-vm/issues/393

Reply:
> Are you fully updated before trying to do this?
No, I normally do not update my malware analysis VMs as vulnerability fixes may break malware functionality I may want to analyse.

I removed some links in https://github.com/mandiant/flare-vm/pull/518/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L28-L36 with the goal to simplify the documentation and leave a link I had tested to work (hoping this will work for other as well and be more straighforward to implement). But of course if this doesn't work, we should add extra documentation. Do you think we need all the links you are adding (thinking of making as easy and straighforward to users as possible).

Reply:
I think it's fair to have fallback links - especially given how annoying Defender can be to disable reliably any tested/helpful resource is appreciated.

Reply:
@0xdf223 all the links you have added seems to be related to disabling Windows Defender, but you put them under _Disable Windows Updates_ (and the link you are removing is also related to this). Am I understanding something wrong?

Reply:
a proper way to disable Defender on new/updated OS versions (including 11) is described [here](https://woshub.com/disable-windows-defender-antivirus/).